### DISCUSSION TITLE
does this work like fbchat library?

### DISCUSSION THREAD

**Discussion by VengeTH** [iTeam-S/Ampalibe#111] [Q&A]
does this work like fbchat library? if so, I am planning to use this to study python to fix the fb-anti-unsend-master by kapi2289 https://github.com/kapi2289/fb-anti-unsend, please do tell me because i need the fb-anti-unsend for all my friends who are always unsending messages

**Comment by gaetan1903** [comment]
Hello, 

no, fbchat is to use entire personnal facebook account, this is special for page messaging that use official facebook api
